translation.rst

```diff
@@ -329,6 +326,44 @@ The translator service is accessible in PHP templates through the
         array('%count%' => 10)
     ) ?>
 
+Extracting Translation Contents and Updating Catalogs Automatically
+-------------------------------------------------------------------
+
+The most time-consuming task when translating an application is to extract all
+the template contents to be translated and to keep all the translation files in
+sync. Symfony includes a command called ``translation:update`` that helps you in
+these tasks.
+
+The "safe mode" of this command uses the ``--dump-messages`` option to output
+in the command console the strings to be translated:
```

Comment:
[...] to output the strings to be translated:

Reply:
I think we should remove this - let's just show the commands below